enforcer/src/main/java/org/wso2/micro/gateway/enforcer/security/jwt/JWTUtil.java

```diff
@@ -131,5 +154,98 @@ public static boolean verifyTokenSignature(SignedJWT jwt, String alias) throws M
         }
     }
 
+    public static PrivateKey getPrivateKey() {
+        PrivateKey privateKey = null;
+        try {
+            String strKeyPEM = "";
+            BufferedReader br = new BufferedReader(new FileReader(ConfigHolder.getInstance().getConfig().
+                    getPrivateKeyPath()));
+            String line;
+            while ((line = br.readLine()) != null) {
+                strKeyPEM += line + "\n";
+            }
+            br.close();
+            strKeyPEM = strKeyPEM.replace(Constants.BEGINING_OF_PRIVATE_KEY, "");
+            strKeyPEM = strKeyPEM.replaceAll(System.lineSeparator(), "");
+            strKeyPEM = strKeyPEM.replace(Constants.END_OF_PRIVATE_KEY, "");
+
+            byte[] encoded = Base64.getDecoder().decode(strKeyPEM);
+            KeyFactory kf = KeyFactory.getInstance(Constants.RSA);
+            PKCS8EncodedKeySpec keySpec = new PKCS8EncodedKeySpec(encoded);
+            RSAPrivateKey rsaPrivateKey = (RSAPrivateKey) kf.generatePrivate(keySpec);
+            privateKey = (PrivateKey) rsaPrivateKey;
+        } catch (IOException | NoSuchAlgorithmException | InvalidKeySpecException e) {
+            log.debug("Error obtaining private key", e);
```

Comment:
Why not throwing the error from here? IMO from the util method we should throw the error to caller. Caller should decide what to do with the error.